modules/core/core-content/src/main/java/com/enonic/xp/core/impl/content/ProjectContentEventListener.java

```diff
@@ -30,10 +30,10 @@
 import com.enonic.xp.security.auth.AuthenticationInfo;
 
 @Component(immediate = true)
-public class ProjectNodeEventListener
+public class ProjectContentEventListener
     implements EventListener
 {
-    private final static Logger LOG = LoggerFactory.getLogger( ProjectNodeEventListener.class );
+    private final static Logger LOG = LoggerFactory.getLogger( ProjectContentEventListener.class );
```

Comment:
Codacy found an issue: ['static' modifier out of order with the JLS suggestions.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6117594)